web/app/js/components/Navigation.jsx

```diff
@@ -119,6 +115,9 @@ const styles = theme => {
       paddingLeft: `${contentPadding}px`,
       paddingRight: `${contentPadding}px`,
     },
+    helpMenuItem: {
+      outline: "none",
```

Comment:
ahhhh one last thing I missed, can we also add a `cursor: "pointer"` here?

Reply:
We chatted IRL and couldn't reproduce this, will make sure to doublecheck `cursor` appearance when tackling #2325